### PR TITLE
15 update docs

### PR DESCRIPTION
**Pull request by rstens**
As per request. First kick at the can.

### CODE REVIEW

**Review by WadeBarnes — APPROVED** [review]
Looks good to me.

**Comment by WadeBarnes** [comment]
@wenzowski, Anything to add?

**Review by wenzowski — COMMENTED** [review]
Would be fab if we could point out that [automatic analysis](https://sonarcloud.io/documentation/analysis/automatic-analysis/) with sonarcloud gets us some scanning quickly, and that full-fleged [SonarScanner](https://sonarcloud.io/documentation/analysis/scan/overview/) is still available regardless of the choice to run on-prem SonarQube Server or SonarCloud.

**Comment by WadeBarnes** [comment]
@wenzowski, Are you able to add wording to that effect?

File: README.md [hunk 1]
```diff
@@ -96,56 +108,118 @@ An example Jenkins file [SonarQube-StaticScan-Jenkinsfile](./jenkins/SonarQube-S
...
+#sonar.test.inclusions=
+
+# Source encoding
+sonar.sourceEncoding=UTF-8
+
+# Exclusions for copy-paste detection
+#sonar.cpd.exclusions=
+```
+
+5. Ask the Developer Experience group to add your project (bcgov is the owner of your project and Dev.Experience manages the bcgov account )
```
Comment (ShellyXueHan) [inline README.md]: New request type for sonarcloud integration is available: https://github.com/BCDevOps/devops-requests

**Review by ShellyXueHan — COMMENTED** [review]
@rstens @WadeBarnes @wenzowski Hi team! We've got the new request type for SonarCloud integration here: https://github.com/BCDevOps/devops-requests Please feel free to include this in the readme.

**Review by WadeBarnes — APPROVED** [review]

**Pull request merged**